res/controllers/Behringer-DDM4000-scripts.js

```diff
@@ -401,11 +400,11 @@ var DDM4000 = new behringer.extension.GenericMidiController({
                         {options: {midi: [note, 0x2A],    key: null, sendShifted: true}}, // Crossfader: Bounce to MIDI Clock
                         {options: {midi: [note, 0x2B],  inKey: null}}, // Crossfader: Beat (Left)
                         {options: {midi: [note, 0x2C],  inKey: null}}, // Crossfader: Beat (Right)
-                        {options: {midi: [cc,   0x2B], outKey: null}}, // Crossfader: Beat 1
-                        {options: {midi: [cc,   0x2C], outKey: null}}, // Crossfader: Beat 2
-                        {options: {midi: [cc,   0x2D], outKey: null}}, // Crossfader: Beat 4
-                        {options: {midi: [cc,   0x2E], outKey: null}}, // Crossfader: Beat 8
-                        {options: {midi: [cc,   0x2F], outKey: null}}, // Crossfader: Beat 16
+                        {options: {midi: [cc,   0x2B], outKey: undefined}}, // Crossfader: Beat 1
+                        {options: {midi: [cc,   0x2C], outKey: undefined}}, // Crossfader: Beat 2
+                        {options: {midi: [cc,   0x2D], outKey: undefined}}, // Crossfader: Beat 4
+                        {options: {midi: [cc,   0x2E], outKey: undefined}}, // Crossfader: Beat 8
+                        {options: {midi: [cc,   0x2F], outKey: undefined}}, // Crossfader: Beat 16
```

Comment:
alternatively you could also leave out the `outKey` completely.

Reply:
Thanks for your quick feedback! I prefer keeping the `outKey` to signal that this control is a button without LED (for users interested in customizing).

Reply:
yeah, makes sense.